core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -28,11 +27,10 @@ import spray.can.Http
 import spray.http.StatusCodes
 import spray.json._
 
-import scala.concurrent.duration._
-import scala.util.{Try, Failure, Success}
-
-import scala.concurrent.ExecutionContext.Implicits.global
 import scala.collection.JavaConversions._
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
While cleaning up imports I noticed that we're using the global execution context. I was worried that it was also being used for slick blocking calls, but thankfully I see a separate thread pool for slick.

This is probably fine for our case, but just fyi another way to do it is to define dispatchers within akka and reference them using the `ActorSystem`. This allows you to define & tune all execution context from within akka.

http://doc.akka.io/docs/akka/current/scala/dispatchers.html